modules/help.nf

```diff
@@ -0,0 +1,71 @@
+
+    Illumina Pipeline Arguments:
+        Mandatory:
+            --directory [path]          Path to directory containing paired fastq files to dehost
+        
+        Optional:
+            --composite_bwa_index [path]    Path to directory containing BWA indexed composite genome
+            --keep_min_map_quality [i]      Integer minimum mapping quality of COVID-19 reads to keep (Default: 60)
+            --remove_min_map_quality [i]    Integer minimum mapping quality of HUMAN reads to remove (Default: 0)
+
+        Example Command:
+            nextflow run phac-nml/ncov-dehoster --human_ref ./hg38.fa -profile conda --illumina --directory ./paired_fastqs
+
+    Nanostripper Pipeline Arguments:
+      **NOTE: I DO NOT RECOMMEND RUNNING THIS PIPELINE AT THE MOMENT, USE THE MINIMAP2 ONE THAT FOLLOWS**
+        Mandatory:
+            --run_name [str]                Seperate results based on a run name input
+            --fast5_directory [path]        Path to barcoded fast5 directories
+            --nanostripper_tool_path [path] Path to nanostripper tool itself        
+            --nanostripper_env_path [path]  Path to nanostripper conda environment (Has to be manually installed)
+            --guppyCPU [path]               Path to guppy CPU conda environment to demultiplex (Proprietary, has to be manually installed)
+            --guppyGPU [path]               Path to guppy GPU conda environment to basecall
+
+        Optional:
+            --min_length [i]                Minimum length of fastq reads to keep (Default: 400)
+            --max_length [i]                Maximum length of fastq reads to keep (Default: 2400)
+            --human_minimap2_index [file]   Path to minimap2 human indexed .mmi file to speed up analysis
+
+        Example Command:
+            nextflow run phac-nml/ncov-dehoster --human_ref ./hg38.fa -profile conda --nanopore --nanostripper
+                --fast5_directory ./fast5_pass/
+                --nanostripper_tool_path ./nanostripper
+                --nanostripper_env_path ./conda/nanostripper_env
+                --guppyCPU ./conda/guppyCPU_env/
+                --guppyGPU ./conda/guppyGPU_env/
+
+    Minimap2 Pipeline Arguments:
+        Mandatory:
+            --fastq_directory [path]            Path to directory containing either barcoded directories OR individual fastq files
+
+        Optional:
+            --fast5_directory [path]            Path to fast5 directories associated with the data
+            --composite_minimap2_index [file]   Path to composite minimap2 .mmi file to speed up analysis
+            --min_length [i]                    Minimum length of fastq reads to keep (Default: 400)
+            --max_length [i]                    Maximum length of fastq reads to keep (Default: 2400)
+
+        Example Command:
+            nextflow run phac-nml/ncov-dehoster --human_ref ./hg38.fa -profile conda --nanopore --minimap2 --fastq_directory ./fastqs
```

Comment:
just for the sake of example consistency, you could adjust hg38.fa to be .fasta as in line 7 here

Reply:
Changed all to .fa